app/allContacts/page.js

```diff
@@ -0,0 +1,55 @@
+"use client";
+import styles from "../page.module.css";
+import { ContactsAPI } from "../contactsAPI/contactsAPI";
+import Link from "next/link";
+
+export default function AllContacts() {
+  const contacts = ContactsAPI.all();
+  return (
+    <main>
+      <div>
+        <h1 style={{ textAlign: "center" }}>All Contacts</h1>
+        <br />
+        <br />
+        <button className={styles.addButton}>
+          <Link href={"/addNew"}>Add Contact</Link>
+        </button>
+        <table className={styles.table}>
+          <thead>
+            <tr>
+              <th>Profile Picture</th>
+              <th>Name</th>
+              <th>Email</th>
+              <th>Phone</th>
+            </tr>
+          </thead>
+          <tbody>
+            {contacts.map((contact) => {
+              return (
+                <tr>
```

Comment:
Error: Missing "key" prop for element in iterator  react/jsx-key
this is were you need to add key